src/storage/btree.c

```diff
@@ -18721,7 +18721,16 @@ btree_compare_key (DB_VALUE * key1, DB_VALUE * key2, TP_DOMAIN * key_domain, int
 
       if (are_types_comparable)
 	{
-	  c = key_domain->type->cmpval (key1, key2, do_coercion, total_order, NULL, key_domain->collation_id);
+	  bool ignore_trailing_space = prm_get_bool_value (PRM_ID_IGNORE_TRAILING_SPACE);
```

Comment:
Since btree comparisons is invoked very often, it doesn't seem like a good way to put it in this.

Reply:
This code was excluded at the next version